server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java

```diff
@@ -293,7 +294,7 @@ void failedEventShouldBeStoredInDeadLetterUnderItsGroupId(RetryEventsListener re
 
         when()
             .get(EventDeadLettersRoutes.BASE_PATH + "/groups/" + GROUP_ID)
-        .then()
+            .then()
```

Comment:
Not sure it is better for readable?
Usually unindent before `given`, `when`, `then`

Reply:
For me personally, in this case, indentation does not have a significant impact on readability. But, if it is necessary for the project and the team, I can try to fix it. It is advisable to add a rule for this case in checkstyle.xml. Should I do this?

Reply:
+1